tests/reference/runtime-test_dict15-6f3af0d.stderr

```diff
@@ -1 +0,0 @@
-KeyError: The dict does not contain the specified key
```

Comment:
Unrelated: These reference outputs got removed. They were added in https://github.com/lcompilers/lpython/pull/2261/files. In that PR, although we added the reference output, it seems we missed to enable those tests in `tests.toml`.

Reply:
Ooo, I see. I've reverted the changes and enabled those tests in `tests.toml`. Please let me know if it sounds good to you. Thanks!